odl/solvers/nonsmooth/proximal_operators.py

```diff
@@ -1406,6 +1407,77 @@ def _call(self, x, out):
     return ProximalConvexConjKLCrossEntropy
 
 
+def proximal_huber(space, gamma):
+    """Proximal factory of the Huber norm.
+
+    Parameters
+    ----------
+    space : `FnBase`
+        Space X which is the domain of the functional F
```

Comment:
X and F don't refer to anything. I'd go for "Domain of the functional."

Reply:
Too much copy-pasting :)

Reply:
Yeah we sure have some cleanup to do in the older parts.